wiki/Katu/en.md

```diff
@@ -9,6 +8,56 @@ tags:
 
 *See also: [Geki](/wiki/Geki)*
 
-**Katu (喝)**, *Katsu*, or *Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset without achieving the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note. This idea originates from the Nintendo DS game Elite Beat Agents which is one of the games the [osu!standard](/wiki/Game_mode/osu!) [game mode](/wiki/Game_mode) is based on.
+*Not to be confused with osu!taiko's Katu*
 
-<!-- TODO: Add links -->
+**Katu (喝)**, *Katsu*, or *Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset without achieving the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note.
+However, this can't be achieved by having a 50 or lower in the combo itself.
+
+There are two kind of Katu, which is based on the combo finishing judgment. A 300-ended Katu rewards more health bonus than 100-ended Katu, but always worse than a Geki.
+
+This idea originates from the Nintendo DS game Elite Beat Agents which is one of the games the [osu!standard](/wiki/Game_mode/osu!) [game mode](/wiki/Game_mode) is based on.
+
+## Screenshots
+
+In here provided the screenshots of obtaining a Katu/Katsu/Beat from Elite Beat Agents, Ouendan 2, and osu! itself.
```

Comment:
this sentence is unnecessary, please add captions to every image instead, something like this:
```markdown
![](img/eba-without-100k.jpg "Katu in Elite Beat Agents")
```